spikeinterface/comparison/hybridspikesrecording.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Union
+import numpy as np
+from spikeinterface.core import BaseSorting, NumpySorting, WaveformExtractor, AddTemplatesRecording
+
+
+class HybridSpikesRecording(AddTemplatesRecording):
+
+	def __init__(self, wvf_extractor: WaveformExtractor, max_n_spikes: int = 1000,
+				 max_frac_spikes: float = 0.05, refractory_period: float = 1.5) -> None:
```

Comment:
`max_n_spikes` -> `max_injected_per_unit` (in line with other args in the API)
`max_frac_spikes` -> `injected_rate`

Reply:
Done.